content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
+depending on the server that is being used. A user can override the start
+command by using the [Paketo Procfile buildpack][bp/procfile]. Check out the
+[Procfiles]({{< ref "/docs/howto/configuration#procfiles" >}}) section
+of our configuration docs for details on how to use the buildpack.
+
+## Enable a session handler via service bindings
+The PHP Buildpack supports using session handlers for
+[Memcached](https://memcached.org/) and [Redis](https://redis.io/) instances
+via the [PHP Memcached Session Handler Buildpack][bp/memcached] and the [PHP
+Redis Session Handler Buildpack][bp/redis].
+Check out the [PHP session handler documentation][external/php-session-handler]
+for more information on what a PHP session handler is. 
 
-  # directory where library code is stored
-  # default: lib
-  libdirectory: lib
+In order to configure a session handler for either Redis or Memcached the user
+must provide a [service binding]({{< ref "/docs/howto/configuration#bindings"
+>}}).
 
-  # cli script to use
-  # no default
-  script:
+### Enable the session handler for Redis
+To configure a Redis instance session handler, the provided service binding
+should contain the following:
 
-  # default: admin@localhost
-  serveradmin: admin@localhost
+| Binding File | Value |  Required | Description |
+| -------- | -------- | -------- | ----------- |
+| `type`   | `php-redis-session` | yes     | Binding type
+| `host` or `hostname`   | Default: 127.0.0.1 | no     | Redis instance IP address
+| `port`  | Default: 6379 | no     | Redis instance port
+| `password` | Omitted if unset | no     | Redis instance password
+
+The configurations from the service binding are parsed and used to create a
+`php-redis.ini` file with session details, and its path is appended to the
+`PHP_INI_SCAN_DIR` for usage by PHP when the app starts up.
+
+When performing a build with the `pack` CLI, passing the service binding will look like the following:
+
+{{< code/copyable >}}
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env SERVICE_BINDING_ROOT=/bindings \
+  --volume <absolute-path-to-binding>:/bindings/php-redis-session
 {{< /code/copyable >}}
 
-## Vendor composer packages
+### Enable the session handler for Memcached
+To configure a Memcached instance session handler, the provided service binding
+should contain the following:
+
+| Binding File | Value |  Required | Description |
+| -------- | -------- | -------- | ----------- |
+| `type`   | `php-memcached-session` | yes     | Binding type
+| `servers`   | Default: 127.0.0.1 | no     | Memcached instance IP address
+| `username`  | Omitted if unset | no     | Memcached instance username
+| `password` | Omitted if unset | no     | Memcached instance password
+
+The configurations from the service binding are parsed and used to create a
+`php-memcached.ini` file with session details, and it's path is appended to the
+`PHP_INI_SCAN_DIR` for usage by PHP when the app starts up.
```

Comment:
Likewise, this might not be relevant for someone who just wants to know how to get things working.
```suggestion
```